lib/cassandra/protocol/v3.rb

```diff
@@ -190,93 +210,117 @@ def actual_decode(buffer, fields, size, code)
 
         def decode_response(opcode, protocol_version, buffer, size, trace_id)
```

Comment:
The majority of this method has formatting changes. I added a few else blocks to some cases. Noted below.